assets/large_language_models/rag/components/validate_deployments/spec.yaml

```diff
@@ -17,7 +17,7 @@ description: |
 inputs:
   embeddings_model:
     type: string
-    optional: true
+    optional: false
```

Comment:
This is optional according to -
https://msdata.visualstudio.com/Vienna/_git/AzureMlCli?path=/src/azureml-rag/src/azureml/rag/tasks/validate_deployments.py&version=GBtest1&line=648&lineEnd=649&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents